omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/DrugResource1_10.java

```diff
@@ -52,8 +60,46 @@ public DelegatingResourceDescription getCreatableProperties() {
 		DelegatingResourceDescription description = super.getCreatableProperties();
 		description.addProperty("strength");
 		description.addRequiredProperty("name");
+		description.addProperty("drugReferenceMaps");
 		
 		return description;
 	}
 	
+	public Model getGETModel(Representation rep) {
+		ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation) {
+			modelImpl
+			        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapGetRef"));
+		} else if (rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapGet"));
+		}
+		return modelImpl;
+	}
+	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		return new ModelImpl()
+		        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapCreate"));
+	}
+	
+	@PropertySetter("drugReferenceMaps")
+	public static void setGroupMembers(Drug drug, List<DrugReferenceMap> members) {
+		if(members == null && members.size() == 0) {
+			return; }
+		for (DrugReferenceMap member : members) {
+			drug.addDrugReferenceMap(member);
+			member.setDrug(drug);
+		}
+
+ 	}
+	
+	@PropertyGetter("drugReferenceMaps")
+	public static Object getGroupMembers(Drug drug) {
```

Comment:
What is the use of this method? Will any test fail without it?

Reply:
ya it was a duplication of the  referencemaps getters conversion in the Drug class